include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -259,24 +259,30 @@ struct transform_reduce
             __local_mem[__local_idx] = __res;
         }
     }
+
+    // For non-SPIR-V targets, we check if the operator is commutative before selecting the appropriate codepath.
+    // On SPIRV targets, we force the non-commutative implementation as this is currently more performant.
+    constexpr static bool
+    use_nonseq_impl()
+    {
+        return !oneapi::dpl::__internal::__is_spirv_target_v && _Commutative::type::value;
+    }
```

Comment:
```suggestion
    static constexpr bool use_nonseq_impl =  !oneapi::dpl::__internal::__is_spirv_target_v && _Commutative::value;
```

Reply:
I agree it is better for this to just be a variable. Changed.